gilbreth_support/src/conveyor_spawner_node.cpp

```diff
@@ -0,0 +1,260 @@
+#include <ros/ros.h>
+#include <xmlrpcpp/XmlRpcValue.h>
+#include <xmlrpcpp/XmlRpcException.h>
+#include <gazebo_msgs/SpawnModel.h>
+#include <std_srvs/Empty.h>
+#include <tf/transform_datatypes.h>
+#include <random>
+
+const static std::string GAZEBO_SPAWN_SERVICE = "gazebo/spawn_urdf_model";
+const static std::string START_SPAWN_SERVICE = "start_spawn";
+const static std::string STOP_SPAWN_SERVICE = "stop_spawn";
+const static double SRV_TIMEOUT = 10.0f;
+
+namespace gilbreth
+{
+namespace simulation
+{
+
+struct SpawnParameters
+{
+  std::vector<std::string> model_topics;
+  std::vector<std::string> model_descriptions;
+  geometry_msgs::Pose initial_pose;
```

Comment:
Would it make sense to have an initial pose per object, this way it'll allow us to avoid situations where one object's geometry may be initially in collision at this pose even though it works for the other objects.